src/System.Net.Http/src/System/Net/Http/Managed/HttpProxyConnectionHandler.cs

```diff
@@ -27,7 +27,7 @@ public HttpProxyConnectionHandler(HttpConnectionSettings settings, HttpMessageHa
             _innerHandler = innerHandler;
             _proxy = settings._proxy ?? new PassthroughWebProxy(s_proxyFromEnvironment.Value);
             _defaultCredentials = settings._defaultProxyCredentials;
-            _connectionPools = new HttpConnectionPools(settings._maxConnectionsPerServer);
+            _connectionPools = new HttpConnectionPools(settings, true, settings._maxConnectionsPerServer);
```

Comment:
Nit: could you name the `true` argument, e.g. `usingProxy: true`, to help make the code more clear?  That should be possible with C# 7.2 now, even though it's not at the end of the parameter list.

Reply:
Sure.  I'll move it to the last param too.